profile/README.md

```diff
@@ -12,20 +12,19 @@ data.
 
 <br clear="right"/>
 
-# Arcaflow
-
+# [Arcaflow](https://arcalot.io/arcaflow)
 <a href="https://arcalot.io/arcaflow"><img align="left" width="200px"
 alt="Arcaflow logo showing a waterfall and a river with 3 trees symbolizing the various
 plugins" src="https://github.com/arcalot/.github/raw/main/branding/arcaflow.svg"></a>
 
-Arcaflow is a workflow system consisting of three main components:
+Arcaflow is a workflow orchestration system consisting of three main components:
 
-* [The Arcaflow engine](https://github.com/arcalot/arcaflow-engine), statically-compiled
-and platform-portable, written in Go
-* [Plugins](https://github.com/orgs/arcalot/repositories?q=arcaflow-plugin-), written in
-various languages and delivered as Linux containers or modules
-* [Workflow definitions](https://github.com/arcalot/arcaflow-workflows), written in
-YAML, tying together plugins and their data
+* [The Arcaflow engine](https://github.com/arcalot/arcaflow-engine) - A
```

Comment:
There's no such thing as a "platform-independent binary" (although there are binary formats including multiple platforms, that's not really the same thing). It's a (theoretically) platform-independent source (maybe, assuming we don't conflict anywhere with Windows by details like assuming "/" path separators ... 😆 ) of which we supply various binaries.